### ISSUE TITLE
[Enhancement] G2 In-page navigation

### ISSUE THREAD

**Issue by jaredhenderson** [unimelb/pattern-lib#1711]
## Currently
The current **In-page navigation** component was built for FFAM and has a clunky and dated design.

![Screen Shot 2021-07-23 at 2.38.03 pm.png](https://images.zenhubusercontent.com/5acae0cd4b5806bc2bce3338/1595acdd-4058-4781-95c5-6fc5baa23fb4)
![Screen Shot 2021-07-23 at 2.38.18 pm.png](https://images.zenhubusercontent.com/5acae0cd4b5806bc2bce3338/3beb90a1-d527-4b58-8334-b52327245556)

[Pattern Library - In-page navigation](https://pattern-lib-unimelb.netlify.app/?selectedKind=Navigation&selectedStory=In%20page%20Navigation&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel)
[CMS Documentation](https://matrix-cms.unimelb.edu.au/web/mce/content-templates/in-page-navigation)

Example page:
https://study.unimelb.edu.au/dev-preview/open-day-2021/open-day-dev/open-day-about-dev/_nocache

Other examples:
https://matrix-cms.unimelb.edu.au/reports/content-templates-reports/in-page-nav

### [Figma designs >](https://www.figma.com/file/UF2630XlnsbudpZgvyZec0/Gen-2-and-3-Master-Components?node-id=2370%3A13712)

![Screen Shot 2021-07-23 at 2.40.08 pm.png](https://images.zenhubusercontent.com/5acae0cd4b5806bc2bce3338/846c4be8-8147-46da-a616-6f41c250c461)
![Screen Shot 2021-07-23 at 2.40.15 pm.png](https://images.zenhubusercontent.com/5acae0cd4b5806bc2bce3338/3ee5382b-d848-40ac-90a4-c40a07eee951)

The new style integrates the new design patterns from Today for the main layout and fixed navigation selector.

- Background colour should be fixed to this light grey so it stays consistent when it changes to fixed position mode
- These colours are consistent with the in-page navigation used on [Microcert pages](https://study.unimelb.edu.au/find/microcredentials/artificial-intelligence-and-women/)
![Screen Shot 2021-07-23 at 2.46.22 pm.png](https://images.zenhubusercontent.com/5acae0cd4b5806bc2bce3338/d0a42209-e9a8-4b8d-bd3a-c49d5dd08496)
- This component needs to be compatible with FFAM styling (no Noto Serif font)

**Comment by gepifanio** [comment]
hey @jaredhenderson I have a question for this one:

I have started this ticket in the PR https://github.com/unimelb/pattern-lib/pull/1737

I am using two main components from Today ( SectionAlt and InPageNavToday )
1. Should I modify the existing styling of the component existing SectionAlt component to match your new design? (Font, sizes, spacing) or should that be a new variation of the current Component?

- For the in-page navigation I used just the current one for now as an example and implementation, I will look into turn into a new variation now to have the dropdown options instead of the bar of links we currently have. [preview](https://deploy-preview-1737--pattern-lib-unimelb.netlify.app/iframe.html?selectedKind=Navigation&selectedStory=In%20page%20Navigation)
